static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -65,6 +65,10 @@ type Props = ReactRouter.WithRouterProps & {
    * Sidebar is at "top" or "left" of screen
    */
   orientation: SidebarOrientation;
+  /**
+   * An optional prefix that can be used to reset the "new" indicator
+   */
+  localStorageNewSuffix?: string;
```

Comment:
how about `isNewCacheBustKey`

Reply:
@EvanPurkhiser I'm open to suggestions but `isNewCacheBustKey` makes it sound like a boolean which its not

Reply:
ah you're right

Reply:
I just don't like that it has this implementation specific name for 'localStorage'

Reply:
I think `isNewCacheBustKey` is OK though since you can see from the type that it's a string and it's associated to the `isNew` prop

Reply:
@EvanPurkhiser ok, how about `cacheSuffix`?

Reply:
I think it needs to be related _somehow_ to the isNew prop

Reply:
I'll go with `isNewCacheBustKey` since TS makes it impossible for someone to accidentally put in a boolean

Reply:
the `isNew` prefix does sound like a boolean, would it be better if we remove the `is`?